ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/PollableSubscriber.java

```diff
@@ -55,6 +56,7 @@ protected void handleValue(final Row row) {
 
   @Override
   protected void handleError(final Throwable t) {
+    failed = true;
     errorHandler.accept(new Exception(t));
```

Comment:
Why wrap it in an exception?

Reply:
Previously it was being passed into a handler that required an Exception rather than a Throwable but this is no longer the case so I've removed this.

Out of curiosity, do you know why reactive streams methods expect Throwables for errors, rather than Exceptions? Most other APIs I've seen work with Exceptions.